src/ZulipMobile.js

```diff
@@ -16,14 +13,6 @@ export default class ZulipMobile extends Component {
 
   componentWillMount() {
     restore(() => {
-      store.subscribe(() => {
```

Comment:
You are right; I reworked it to be called both on componentDidMount and shouldComponentUpdate. (first is called once, second is called every time but the first)